native-maven-plugin/src/main/java/org/graalvm/buildtools/maven/config/MetadataCopyConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package org.graalvm.buildtools.maven.config;
+
+import org.apache.maven.plugins.annotations.Parameter;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class MetadataCopyConfiguration {
+
+    @Parameter
+    private List<Path> outputDirectories;
+
+    @Parameter(defaultValue = "${project.basedir}/src/main/resources/META-INF/native-image")
+    private Path outputDirectory;
```

Comment:
Please add javadoc since that goes into the generated `plugin.xml` and shows in the autocomplete documentation for users.